### PR TITLE
android: bump versions of android test dependencies

### PR DESCRIPTION
**Pull request by Augustyniak**
Description: Update the versions of Android test dependencies.
Risk Level: Low, test impacts test dependencies only
Testing: CI
Docs Changes: N/A 
Release Notes: N/A

Signed-off-by: Rafal Augustyniak <raugustyniak@lyft.com>

### CODE REVIEW

File: test/java/org/chromium/net/BidirectionalStreamTest.java
Comment (alyssawilk) [inline test/java/org/chromium/net/BidirectionalStreamTest.java:1547]: @RyanTheOptimist I'm good commenting these out if you are.  I hate to have upgrades blocked on it.

**Review by alyssawilk — APPROVED** [review]

File: test/java/org/chromium/net/BidirectionalStreamTest.java
Comment (RyanTheOptimist) [reply test/java/org/chromium/net/BidirectionalStreamTest.java:1547]: I think this sounds reasonable, but we should probably file an issue about this so we don't lose track of it. Maybe @colibie has thoughts on how to resolve the missing symbol issues.

Comment (colibie) [reply test/java/org/chromium/net/BidirectionalStreamTest.java:1547]: Turns out the previous impl was using the symbols from the cronet in the android platform. Will fix this with [PR2568](https://github.com/envoyproxy/envoy-mobile/pull/2568)

Comment (Augustyniak) [reply test/java/org/chromium/net/BidirectionalStreamTest.java:1547]: @colibie thank you for the context. I am going to wait with my PR until yours is merged then.

Comment (colibie) [reply test/java/org/chromium/net/BidirectionalStreamTest.java:1547]: You can merge yours, I'll update mine. Still making some changes on it anyway.

**Pull request merged**
